addons/website_event_exhibitor/static/src/js/event_exhibitor_connect.js

```diff
@@ -13,7 +13,7 @@ publicWidget.registry.eventExhibitorConnect = publicWidget.Widget.extend({
      */
     init: function () {
         this._super(...arguments);
-        this._onConnectClick = debounce(this._onConnectClick, 500, true);
+        this._onConnectClick = debounce(this._onConnectClick, 500, true).bind(this);
```

Comment:
Yes, this is what I meant - but either let's do it within a single commit - or only add the event listener aspect inside its own commit.

Reply:
wonderful since changes are minimal, lets make it in one commit then. Thanks!